lib/beetle/configuration.rb

```diff
@@ -73,8 +73,13 @@ class Configuration
 
     # list of amqp servers to use (defaults to <tt>"localhost:5672"</tt>)
     attr_accessor :servers
+
     # list of additional amqp servers to use for subscribers (defaults to <tt>""</tt>)
     attr_accessor :additional_subscription_servers
```

Comment:
TODO: try to add aznmq server to this list and run e2e test. Make sure that this is covered by the unit test

Reply:
We did that and it works.